content/en/docs/tasks/job/pod-failure-policy.md

```diff
@@ -28,9 +28,6 @@ You should already be familiar with the basic use of [Job](/docs/concepts/worklo
 
 {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
 
-Ensure that the [feature gates](/docs/reference/command-line-tools-reference/feature-gates/)
-`PodDisruptionConditions` and `JobPodFailurePolicy` are both enabled in your cluster.
-
 ## Using Pod failure policy to avoid unnecessary Pod retries
 
 With the following example, you can learn how to use Pod failure policy to
```

Comment:
Sure, I'm mentioning this also in the other sort of related PR https://github.com/kubernetes/website/pull/46808. Maybe a complete list of scenarios in which the condition is added would be helpful?

Reply:
I see, I missed that PR. I think we do not have to combine the PRs - we can have the main explanation in https://github.com/kubernetes/website/pull/46808 and have small bits here if needed. 

> Maybe a complete list of scenarios in which the condition is added would be helpful?

I will leave that up to you. But maybe we could at least mention/link the PodFailurePolicy and SuccessPolicy there.

Reply:
In that case I think I prefer to have two PRs, keeping this minimal, because there were actually no functional changes to Pod Failure Policy in this release cycle.